aspnetcore/fundamentals/websockets.md

```diff
@@ -92,6 +92,32 @@ The WebSocket closed exception can also happen when returning too soon from an a
 
 Never use `Task.Wait`, `Task.Result`, or similar blocking calls to wait for the socket to complete, as that can cause serious threading issues. Always use `await`.
 
+::: moniker range=">= aspnetcore-6.0"
+
+### Using Compression
```

Comment:
We avoid gerunds in headings because they don't translate well. Can this be? ...

```suggestion
### Compression
```

Reply:
Or `Use compression"
Also, sentence casing.